medicines/web/src/components/cookie-form/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import Cookies from 'universal-cookie';
+
+import { baseSpace } from '../../styles/dimensions';
+
+const StyledCookieForm = styled.section`
+  label {
+    padding: ${baseSpace};
+    padding-right: calc(${baseSpace} + 10px);
+    input {
+      margin-right: 30px;
+    }
+  }
+`;
+
+interface ICookieForm {
+  storageAllowed: boolean;
+  setStorageAllowed: any;
+}
+
+const CookieForm: React.FC<ICookieForm> = props => {
+  // NOTE: By "cookies", we mean "cookies and similar technologies". This includes
+  // local storage, session storage, etc.
+  const handleCookiesOn = () => {
+    props.setStorageAllowed(true, true);
+  };
+
+  const handleCookiesOff = () => {
+    props.setStorageAllowed(false);
+
+    // Delete *everything*
+    window.localStorage.clear();
+    window.sessionStorage.clear();
+    const cookies = new Cookies();
+    for (const cookieName of Object.keys(cookies.getAll())) {
+      cookies.remove(cookieName);
+    }
+
+    // Reload the page. Seems to be the only way to stop analytics.
+    window.location.href = '/cookies';
```

Comment:
Sometimes, the cookie page will be loaded with the URL `/cookies/` instead of `/cookies`. I don't know why. When this happens, reloading the page using `window.location.refresh()` or `window.history.go(0)` will cause a 404.